### PR TITLE
About Amazon Simple Storage Service, Identity and Access Management, and GovCloud

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.4, OCP 4.14+

Resolves https://issues.redhat.com/browse/OADP-2428 by adding a new section, "About Amazon Simple Storage Service, Identity and Access Management, and GovCloud" to the instructions for installing OADP on AWS.

Preview:https://68995--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

**Review by weshayutin — COMMENTED** [review]
/hold do not merge

**Comment by weshayutin** [comment]
The notes about gov cloud have to be removed until we confirm from eng/QE that gov cloud works

**Comment by RichardHoch** [comment]
@anarnold97 As we discussed, this PR is "on ice" until QE verifies our GovCloud capabilities.

**Comment by kalexand-rh** [comment]
4.11 is now [EOL](https://access.redhat.com/support/policy/updates/openshift), and we are no longer updating it. Please set the versions to 4.12+.

**Review by PrasadJoshi12 — APPROVED** [review]

File: modules/oadp-s3-and-gov-cloud.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-s3-and-gov-cloud_{context}"]
+= About Amazon Simple Storage Service, Identity and Access Management, and GovCloud
+
+Amazon Simple Storage Service (Amazon S3) is Amazon's storage solution for the internet. As an authorized user, you can use this service to store and retrieve any amount of data whenever you want, from anywhere on the web.
+
+You securely control access to Amazon S3 and other Amazon services by using the AWS Identity and Access Management (IAM) web service.
+
+You can use IAM to manage permissions that control which AWS resources users can access. You use IAM to both authenticate - verify that a user is who they claim to be -  and to authorize - grant permissions to use resources.
```
Comment (mburke5678) [inline modules/oadp-s3-and-gov-cloud.adoc:13]: Do not use em dashes in technical docs, per ISG. Maybe:
```suggestion
You can use IAM to manage permissions that control which AWS resources users can access. You use IAM to both authenticate, or verify that a user is who they claim to be, and to authorize, or grant permissions to use resources.
```

File: modules/oadp-s3-and-gov-cloud.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+** In AWS GovCloud (US) Regions, ARNs have an identifier that is different from the one in other standard AWS Regions, `arn:aws-us-gov`.  If an ARN requires you to specify a Region, US-West or US-East, use one the following ARNs:
+
+*** For US-West, use `us-gov-west-1`.
+*** For US-East, use `us-gov-east-1`.
+
+
+** For all other standard regions, ARNs begin with: `arn:aws`.
+
+* In AWS GovCloud (US) Regions, use the endpoints listed in the *AWS GovCloud (US-East)* and *AWS GovCloud (US-West)* rows of the "Amazon S3 endpoints" table on link:https://docs.aws.amazon.com/general/latest/gr/s3.html[Amazon Simple Storage Service endpoints and quotas]. If you are processing export-controlled data, use one of the SSL/TLS endpoints. If you have FIPS requirements, use a FIPS 140-2 endpoint - \https://s3-fips.us-gov-west-1.amazonaws.com or \https://s3-fips.us-gov-east-1.amazonaws.com.
+* Other AWS-imposed restrictions as found on link: https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-s3.html#govcloud-S3-diffs[How Amazon Simple Storage Service Differs for AWS GovCloud (US)]
```
Comment (mburke5678) [inline modules/oadp-s3-and-gov-cloud.adoc:29]: Make this one a sentence to be parallel with the other single-bullet items, 
```suggestion
* Other AWS-imposed restrictions can be found on link: https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-s3.html#govcloud-S3-diffs[How Amazon Simple Storage Service Differs for AWS GovCloud (US)].
```

**Comment by mburke5678** [comment]
@RichardHoch Can you generate a preview? The preview links above are long dead.

**Comment by RichardHoch** [comment]
@mburke5678 I made the changes you suggested and updated the preview. please review this PR again. Thanks!

**Review by xenolinux — COMMENTED** [review]
Merge review submitted - a few comments. 

Preview needs to be regenerated. The current preview links give an error.

**Comment by RichardHoch** [comment]
@xenolinux Thanks for the review. I made the changes you suggested. Please review the PR again. if good, please merge. Thanks!

File: modules/oadp-s3-and-gov-cloud.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-s3-and-gov-cloud_{context}"]
+= About Amazon Simple Storage Service, Identity and Access Management, and GovCloud
+
+Amazon Simple Storage Service (Amazon S3) is Amazon's storage solution for the internet. As an authorized user, you can use this service to store and retrieve any amount of data whenever you want, from anywhere on the web.
```
Comment (xenolinux) [inline modules/oadp-s3-and-gov-cloud.adoc:9]: Do not use ’s to show the possessive form of abbreviations or product names.
```suggestion
Amazon Simple Storage Service (Amazon S3) is a storage solution of Amazon for the internet. As an authorized user, you can use this service to store and retrieve any amount of data whenever you want, from anywhere on the web.
```

> Do not use ’s to show the possessive form of abbreviations or product names.
Incorrect example: IBM Instana’s capabilities yield real-time monitoring, proactive issue diagnosis, and performance optimization.
Correct example : The capabilities of IBM Instana yield real-time monitoring, proactive issue diagnosis, and performance optimization.

File: modules/oadp-s3-and-gov-cloud.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+** IIn AWS GovCloud (US) regions, ARNs have an identifier that is different from the one in other standard AWS regions, `arn:aws-us-gov`. If you need to specify the US-West or US-East region, use one the following ARNs:
+
+*** For US-West, use `us-gov-west-1`.
+*** For US-East, use `us-gov-east-1`.
+
+
+** For all other standard regions, ARNs begin with: `arn:aws`.
+
+* In AWS GovCloud (US) regions, use the endpoints listed in the *AWS GovCloud (US-East)* and *AWS GovCloud (US-West)* rows of the "Amazon S3 endpoints" table on link:https://docs.aws.amazon.com/general/latest/gr/s3.html[Amazon Simple Storage Service endpoints and quotas]. If you are processing export-controlled data, use one of the SSL/TLS endpoints. If you have FIPS requirements, use a FIPS 140-2 endpoint - \https://s3-fips.us-gov-west-1.amazonaws.com or \https://s3-fips.us-gov-east-1.amazonaws.com.
+* To find the other AWS-imposed restrictions, see: https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-s3.html#govcloud-S3-diffs[How Amazon Simple Storage Service Differs for AWS GovCloud (US)]
```
Comment (xenolinux) [inline modules/oadp-s3-and-gov-cloud.adoc:29]:
```suggestion
* To find the other AWS-imposed restrictions, see https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-s3.html#govcloud-S3-diffs[How Amazon Simple Storage Service Differs for AWS GovCloud (US)].
```

- `:` is not required here so can be removed.
- Full stop is missing.

File: modules/oadp-s3-and-gov-cloud.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+** IIn AWS GovCloud (US) regions, ARNs have an identifier that is different from the one in other standard AWS regions, `arn:aws-us-gov`. If you need to specify the US-West or US-East region, use one the following ARNs:
+
+*** For US-West, use `us-gov-west-1`.
+*** For US-East, use `us-gov-east-1`.
+
+
+** For all other standard regions, ARNs begin with: `arn:aws`.
+
+* In AWS GovCloud (US) regions, use the endpoints listed in the *AWS GovCloud (US-East)* and *AWS GovCloud (US-West)* rows of the "Amazon S3 endpoints" table on link:https://docs.aws.amazon.com/general/latest/gr/s3.html[Amazon Simple Storage Service endpoints and quotas]. If you are processing export-controlled data, use one of the SSL/TLS endpoints. If you have FIPS requirements, use a FIPS 140-2 endpoint - \https://s3-fips.us-gov-west-1.amazonaws.com or \https://s3-fips.us-gov-east-1.amazonaws.com.
```
Comment (xenolinux) [inline modules/oadp-s3-and-gov-cloud.adoc:28]: Links [1] are not currently being rendered in the preview. Are these links supposed to be rendering? 

[1] \https://s3-fips.us-gov-west-1.amazonaws.com or \https://s3-fips.us-gov-east-1.amazonaws.com.

```suggestion
* In AWS GovCloud (US) regions, use the endpoints listed in the *AWS GovCloud (US-East)* and *AWS GovCloud (US-West)* rows of the "Amazon S3 endpoints" table on link:https://docs.aws.amazon.com/general/latest/gr/s3.html[Amazon Simple Storage Service endpoints and quotas]. If you are processing export-controlled data, use one of the SSL/TLS endpoints. If you have FIPS requirements, use a FIPS 140-2 endpoint such as https://s3-fips.us-gov-west-1.amazonaws.com or https://s3-fips.us-gov-east-1.amazonaws.com.
```

File: modules/oadp-s3-and-gov-cloud.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+Amazon Simple Storage Service (Amazon S3) is Amazon's storage solution for the internet. As an authorized user, you can use this service to store and retrieve any amount of data whenever you want, from anywhere on the web.
+
+You securely control access to Amazon S3 and other Amazon services by using the AWS Identity and Access Management (IAM) web service.
+
+You can use IAM to manage permissions that control which AWS resources users can access. You use IAM to both authenticate, or verify that a user is who they claim to be, and to authorize, or grant permissions to use resources.
+
+AWS GovCloud (US) is an Amazon storage solution developed to meet the stringent and specific data security requirements of the United States Federal Government. AWS GovCloud (US) works the same as Amazon S3 except for the following:
+
+* You cannot copy the contents of an Amazon S3 bucket in the AWS GovCloud (US) Regions directly to or from another AWS Region.
```
Comment (xenolinux) [inline modules/oadp-s3-and-gov-cloud.adoc:17]:
```suggestion
* You cannot copy the contents of an Amazon S3 bucket in the AWS GovCloud (US) regions directly to or from another AWS region.
```
I think this change is missed to apply.

**Review by xenolinux — COMMENTED** [review]
After being able to access the generated preview, I have a few more comments and a question about Amazon doc links. Sorry!

**Pull request merged**